migtests/tests/oracle-tests/ora-datatypes-test/env.sh

```diff
@@ -6,4 +6,5 @@ export SOURCE_DB_PASSWORD=${SOURCE_DB_PASSWORD:-"Password1"}
 export SOURCE_DB_SCHEMA=${SOURCE_DB_SCHEMA:-"ADMIN"}
 export TARGET_DB_HOST=${TARGET_DB_HOST:-"10.9.203.12"}
 export TARGET_DB_NAME=${TARGET_DB_NAME:-"datatypes_test"}
-export TARGET_DB_PASSWORD=${TARGET_DB_PASSWORD:-"password"}
\ No newline at end of file
+export TARGET_DB_USER="yugabyte"
+export TARGET_DB_PASSWORD=''
```

Comment:
As I have told earlier, this is not the place to set host, user-name, and password.

Do you see any problem with passing those values from the test execution environment? e.g. .bashrc file on your dev machine or the script triggering the tests automatically.

Reply:
These changes were done to test the changes delivered by @ShivanshGahlot. This PR is still WIP @amit-yb